pkg/api/vm/handler.go

```diff
@@ -1143,7 +1143,7 @@ func (h *vmActionHandler) cloneVM(name string, namespace string, input CloneInpu
 	if err != nil {
 		return fmt.Errorf("cannot get vm %s/%s, err: %w", namespace, name, err)
 	}
-	newVM := getClonedVMYamlFromSourceVM(input.TargetVM, vm)
+	newVM := getClonedVMYamlFromSourceVM(input.TargetVM, vm, kubevirtv1.VirtualMachineRunStrategy(input.RunStrategy))
```

Comment:
Nit: just pass in the entire `input` object?

Reply:
Address it. Thanks.